### ISSUE TITLE
Check for any network connection, not strictly WiFi, when checking for devices

### ISSUE THREAD

**Issue by jcstaudt** [wseemann/RoMote#31]
### Observed behavior
RoMote checks for an active WiFi connection when checking for network-connected devices. My phone is connected to the network via an Ethernet connection; although Roku network devices exist on the local network, RoMote is unusable as there is no clear way to bypass the WiFi notification pop-up.

### Desired behavior
RoMote checks for Roku devices on the local network whether a WiFi or Ethernet connection is established. A connection can succeed through the Ethernet network interface even if the wireless interface is disabled.

**Comment by Mrpuppyz** [comment]
I would really like for this app to allow continuing passed the no wifi network as well. I used tailscale on my wifi router that the Roku device in connected to so my phone can use cellular service but still have access to my local home network. With the manual ip address I can just point the app to my roku though my VPN and not have to turn on my wifi.

**Comment by ajdunevent** [comment]
I just want to +1 this request.  In particular, it would help Linux mobile users who run RoMote in Waydroid.  There aren't any particularly good Roku clients for Linux at the moment and none of them have the option to include Private Listening.

**Comment by dirwiz** [comment]
Yep +1 on this.  Ive run into the same problem with an ethernet connection.

**Comment by wseemann** [comment]
This was added in the latest [release](https://github.com/wseemann/RoMote/releases/tag/v2.0.0).
